### ISSUE TITLE
datetimeoffset type SQL default function getdate() does not translate

### ISSUE THREAD

**Issue by rocker8942** [ReversePOCO/EntityFramework-Reverse-POCO-Code-First-Generator#453]
Default value set on datetimeoffset type is not translated. 

Database as below. 
```sql
CREATE TABLE [dbo].[DateTimeDefaultTest](
	[Id] [int] NOT NULL,
	[CreatedDate] [datetimeoffset](7) NULL
) ON [PRIMARY]

GO

ALTER TABLE [dbo].[DateTimeDefaultTest] ADD  CONSTRAINT [DF_DateTimeDefaultTest_CreatedDate]  DEFAULT (getdate()) FOR [CreatedDate]
GO
```
Translated to 
```c#
    [System.CodeDom.Compiler.GeneratedCode("EF.Reverse.POCO.Generator", "2.31.0.0")]
    public partial class DateTimeDefaultTest
    {
        public int Id { get; set; } // Id (Primary key)
        public System.DateTimeOffset? CreatedDate { get; set; } // CreatedDate

        public DateTimeDefaultTest()
        {
            InitializePartial();
        }

        partial void InitializePartial();
    }
```
It would be great if the constructor can be translated as below. 
```c#
        public DateTimeDefaultTest()
        {
            **CreatedDate = DateTimeOffset.UtcNow;**
            InitializePartial();
        }
```

**Comment by sjh37** [comment]
Thanks. This will be in the next release.

**Comment by rocker8942** [comment]
Thanks, Simon.

**Comment by sjh37** [comment]
Released v2.37.2
